lldb/test/API/functionalities/data-formatter/type-recognizer/value-object-cast/main.cpp

```diff
@@ -0,0 +1,149 @@
+enum class CastTarget {
+  A_1,
+  A_2,
+
+  B_1,
+  B_2,
+  B_3,
+  B_4,
+
+  C_1,
+  C_2,
+  C_3,
+  C_4,
+};
+
+/// Non-Virtual Inheritance:
```

Comment:
Those diagrams explain what's going on in this whole file, so they should be moved to the top of the file